src/app/components/formElements/TextBox/textBox.module.css

```diff
@@ -1,26 +1,15 @@
 .container {
+  display: flex;
+  flex-direction: column;
   padding-top: 10px;
   padding-bottom: 10px;
-}
```

Comment:
Please reuse this style, creating a `formElements.module.css`. The textarea should also use this. If there's something specific that the textarea component container requires, include it in it's own module css file with the only properties it require (or need to override).